src/components/common/Extensions/MainNavigation/MainNavigation.tsx

```diff
@@ -24,6 +25,7 @@ const MainNavigation: FC<MainNavigationProps> = ({
   const isMobile = useMobile();
   const navItems = useGetNavItems(name);
   const nativeToken = useGetNetworkToken();
+  const { setisActionSidebarOpen } = useActionSidebarContext();
```

Comment:
You could create a separate functions to toggle between the action sidebar state:
```
const { setIsActionSidebarOpen, openActionSidebar, closeActionSidebar, toggleActionSidebar } = useActionSidebarContext();
```

It would be easier to use like so:
```
<SomeComponent
  onClick={openActionSidebar}
/>
```